tests/test_backends.py

```diff
@@ -25,7 +30,7 @@ def test_backends_case_sensitive(self):
 
         model.fit(X, y)
 
-        hb_model = hummingbird.ml.convert(model, "pYtOrCh")
+        hb_model = hummingbird.ml.convert(model, "tOrCh")
```

Comment:
Can we also add a test that "pytorch" still works?

Reply:
yep